scripts/actions/utils/related-resources-helpers.js

```diff
@@ -31,6 +35,7 @@ exports.getExcludedUrls = (frontmatter, siteUrl) => {
   return resources
     .map((resource) => resource.url)
     .concat(redirects)
+    .map((url) => appendLeadingSlash(url))
```

Comment:
channeling my inner zack, i think you can do [point free](https://mostly-adequate.gitbook.io/mostly-adequate-guide/ch05#pointfree) syntax here.

this would be then:
```js
.map(appendLeadingSlash)
```

Reply:
oh hello